WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -41,32 +41,61 @@ class WordPressAuthenticationManager: NSObject {
+                                                   textButtonHighlightColor: .primaryDark,
+                                                   instructionColor: .text,
+                                                   subheadlineColor: .textSubtle,
+                                                   placeholderColor: .textPlaceholder,
+                                                   viewControllerBackgroundColor: .listBackground,
+                                                   textFieldBackgroundColor: .listForeground,
+                                                   buttonViewBackgroundColor: .authButtonViewBackground,
+                                                   navBarImage: .gridicon(.mySites),
+                                                   navBarBadgeColor: .accent(.shade20),
+                                                   prologueBackgroundColor: .primary,
+                                                   prologueTitleColor: .textInverted,
+                                                   statusBarStyle: .lightContent)
+
+        let unifiedStyle = WordPressAuthenticatorStyle(primaryNormalBackgroundColor: .primaryButtonBackground,
+                                                       primaryNormalBorderColor: nil,
+                                                       primaryHighlightBackgroundColor: .primaryButtonDownBackground,
+                                                       primaryHighlightBorderColor: nil,
+                                                       secondaryNormalBackgroundColor: .authSecondaryButtonBackground,
+                                                       secondaryNormalBorderColor: .secondaryButtonBorder,
+                                                       secondaryHighlightBackgroundColor: .secondaryButtonDownBackground,
+                                                       secondaryHighlightBorderColor: .secondaryButtonDownBorder,
+                                                       disabledBackgroundColor: .textInverted,
+                                                       disabledBorderColor: .neutral(.shade10),
+                                                       primaryTitleColor: .white,
+                                                       secondaryTitleColor: .text,
+                                                       disabledTitleColor: .neutral(.shade20),
+                                                       textButtonColor: .primary,
+                                                       textButtonHighlightColor: .primaryDark,
+                                                       instructionColor: .text,
+                                                       subheadlineColor: .textSubtle,
+                                                       placeholderColor: .textPlaceholder,
+                                                       viewControllerBackgroundColor: .white,
```

Comment:
Since the only difference between the two styles is 
`viewControllerBackgroundColor: .listBackground` vs
`viewControllerBackgroundColor: .white`
would it be possible to use the ternary condition in the initializer instead of initializing `WordPressAuthenticatorStyle` twice?

Reply:
I like it! It's a smaller footprint, especially when I don't know how many colors I will need to change.